tasks/testwasher.py

```diff
@@ -148,3 +153,61 @@ def get_tests_family(self, test_name_list):
                 test_name = test_name.rsplit('/', 1)[0]
                 test_family.add(test_name)
         return test_family
+
+
+@task()
+def generate_flake_finder_pipeline(_, n=3):
+    """
+    Verify that the jobs defined within job_files contain a change path rule.
+    """
+
+    # Read gitlab config
+    config = generate_gitlab_full_configuration(".gitlab-ci.yml", {}, return_dump=False, apply_postprocessing=True)
+
+    for job in config:
+        print(job)
+    # Lets keep only variables and jobs with flake finder variable
+    kept_job = {}
+    for job, job_details in config.items():
+        if (
+            'variables' in job_details
+            and 'FLAKES_FINDER' in job_details['variables']
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
I suggest renaming it to `SHOULD_RUN_IN_FLAKES_FINDER`

Reply:
Looks good!